src/frontend/sharedComponents/CameraView.tsx

```diff
@@ -85,6 +85,7 @@ export const CameraView = ({onAddPress}: Props) => {
     ref.current
       .takePictureAsync(captureOptions)
       .then(pic => {
+        if (!pic) return;
```

Comment:
not really sure when this happens in practice, but guessing it's fine to just do an early return in this case.